### ISSUE TITLE
Remove CoreXT workaround added due to lack of support for some CoreFX packages

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#13204]
Once CoreXT updates all branches to NuGet 3.5 remove a workaround in DevDivInsertionFiles that blocks automatic insertion of CoreXT incompatible packages.

In default.config file, remove the `CoreFX.` prefix from the following entries:

```
    <package id="CoreFX.System.Diagnostics.Process" version="4.1.0" link="src\ExternalAPIs\System.Diagnostics.Process" tags="exapis" />
    <package id="CoreFX.System.AppContext" version="4.1.0" link="src\ExternalAPIs\System.AppContext" tags="exapis" />
    <package id="CoreFX.System.Security.Cryptography.X509Certificates" version="4.1.0" link="src\ExternalAPIs\System.Security.Cryptography.X509Certificates" tags="exapis" />
    <package id="CoreFX.System.Security.Cryptography.Algorithms" version="4.2.0" link="src\ExternalAPIs\System.Security.Cryptography.Algorithms" tags="exapis" />
    <package id="CoreFX.System.Reflection.TypeExtensions" version="4.1.0" link="src\ExternalAPIs\System.Reflection.TypeExtensions" tags="exapis" />
    <package id="CoreFX.System.Net.Security" version="4.0.0" link="src\ExternalAPIs\System.Net.Security" tags="exapis" />  
    <package id="CoreFX.System.IO.Compression" version="4.1.0" link="src\ExternalAPIs\System.IO.Compression" tags="exapis" /> 
```

The packages were generated from `%internal_share%\public\tomat\corext` and manually inserted.

In `src\Setup\DevDivInsertionFiles\BuildDevDivInsertionFiles.vb` remove `IsCoreXTCompatible` property.

**Comment by jasonmalinowski** [comment]
@tmat: looks like you already did this?

**Comment by tmat** [comment]
Fixed by #25416
